autoware_lanelet2_extension/test/src/test_utilities.cpp

```diff
@@ -158,6 +158,15 @@ TEST_F(TestSuite, OverwriteLaneletsCenterline)  // NOLINT for gtest
   }
 }
 
+TEST_F(TestSuite, GetLaneletLength)  // NOLINT for gtest
+{
+  double length_2d = lanelet::utils::getLaneletLength2d(road_lanelet);
+  double length_3d = lanelet::utils::getLaneletLength3d(road_lanelet);
+
+  EXPECT_GT(length_2d, 0.0);
+  EXPECT_GT(length_3d, 0.0);
```

Comment:
>     p1 = Point3d(getId(), 0., 0., 0.);
>     p2 = Point3d(getId(), 0., 1., 0.);
>     p3 = Point3d(getId(), 1., 0., 0.);
>     p4 = Point3d(getId(), 1., 1., 0.);
>     LineString3d ls_left(getId(), {p1, p2});
>     LineString3d ls_right(getId(), {p3, p4});

Despite these settings, are they strange that the lengths are zero?

Reply:
The last time was using GT (greater than).
The exact values compared in fd97b01e67e2a3a537580b5961dec70798a2eb06